public/app/features/logs/components/logParser.test.ts

```diff
@@ -12,6 +12,7 @@ describe('logParser', () => {
         dataFrame: new MutableDataFrame({
           refId: 'A',
           fields: [
+            testLineField,
```

Comment:
now (correctly), we ignore the first string-field in the dataframe (which is the log-line-field). to make the current tests work, i add an explicit log-line-field, which will get removed.